### PR TITLE
Use the apache certs

### PR DESCRIPTION
**Pull request by ekohl**
@stbenjam noticed we serve incorrect certificates in development.

### CODE REVIEW

File: manifests/apache.pp [hunk 1]
```diff
@@ -18,16 +18,16 @@
...
     port                => 443,
     priority            => '05',
     options             => ['SymLinksIfOwnerMatch'],
     ssl                 => true,
-    ssl_cert            => $certs::ca_cert,
-    ssl_key             => $certs::ca_key,
-    ssl_ca              => $certs::ca_cert,
+    ssl_cert            => $certs::apache::server_cert,
+    ssl_key             => $certs::apache::server_key,
+    ssl_ca              => $certs::apache::default_ca,
```
Comment (stbenjam) [inline manifests/apache.pp]: Did you try this? When I tried the installer last week with similar settings, puppet didn't have the values for ::certs::apache::server_cert and server_key for some reason.

File: templates/env.erb [hunk 2]
```diff
@@ -1,3 +1,3 @@
 BIND=0.0.0.0
 PORT=<%= scope['katello_devel::rails_port'] %>
-WEBPACK_OPTS='--https --key <%= scope['certs::ca_key'] %> --cert <%= scope['certs::ca_cert'] %> --cacert <%= scope['certs::ca_cert'] %> --host 0.0.0.0 --public <%= @fqdn %>'
+WEBPACK_OPTS='--https --key <%= scope['certs::apache::server_key'] %> --cert <%= scope['certs::apache::server_cert'] %> --cacert <%= scope['certs::apache::default_ca'] %> --host 0.0.0.0 --public <%= scope['certs::apache::hostname'] %>'
```
Comment (stbenjam) [inline templates/env.erb]: Did you try this? When I tried the installer last week with similar settings, puppet didn't have the values for ::certs::apache::server_cert and server_key for some reason.  Maybe referencing the values in apache.pp fixed it, I didn't have that.

Comment (ekohl) [reply templates/env.erb]: I didn't (note the `WIP`). There is a `certs::apache` class definition before any config takes place so in theory ... :)

Comment (ekohl) [reply templates/env.erb]: Updated and now with spec tests

**Comment by ekohl** [comment]
Rebased

**Pull request merged**